File: RSPAClient/Core/AtPar.ViewModel/VM_POU_GETCASEREVIEW_RPT_CASE_ITEM_INFO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_POU_GETCASEREVIEW_RPT_CASE_ITEM_INFO
    {
        public string ITEM { get; set; }
        public string ITEM_DESCRIPTION { get; set; }
        public double? ITEM_COST { get; set; }
        public double? PICKED { get; set; }
        public double? ISSUED_DURING_PROCEDURE { get; set; }
        public double? RETURNED { get; set; }
        public double? WASTED { get; set; }
        public double? CONSUMED { get; set; }
        public double? CONSUMED_COST { get; set; }

    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_POU_GETCASEREVIEW_RPT_CASE_ITEM_INFO.ts
export class VM_POU_GETCASEREVIEW_RPT_CASE_ITEM_INFO {
        public ITEM :string
        public ITEM_DESCRIPTION :string
        public ITEM_COST :any
        public PICKED :any
        public ISSUED_DURING_PROCEDURE :any
        public RETURNED :any
        public WASTED :any
        public CONSUMED :any
        public CONSUMED_COST: any
        public ITEMDETAILS: any[] = [];

        public CHILDPICKEDTOTAL: any=0;
        public CHILDISSUEDTOTAL: any=0;
        public CHILDRETURNEDTOTAL: any=0;
        public CHILDWASTEDTOTAL: any=0;
        public CHILDCONSUMEDTOTAL: any=0;

    }